Tests/FluentAssertions.Specs/Primitives/ObjectAssertionSpecs.cs

```diff
@@ -1290,3 +1364,51 @@ public void Dispose()
         // Ignore
     }
 }
+
+internal class SomeClass
```

Comment:
🔧 We prefer to keep the test classes as simple and specific for the individual test cases as possible, and also next to them (in the same grouping class, e.g. `BeOneOf`. See also